docs/ddl-operation-on-carbondata.md

```diff
@@ -27,18 +27,19 @@ The following DDL operations are supported in CarbonData :
 * [SHOW TABLE](#show-table)
 * [DROP TABLE](#drop-table)
 * [COMPACTION](#compaction)
+* [BUCKETING](#bucketing)
 
 ## CREATE TABLE
   This command can be used to create a CarbonData table by specifying the list of fields along with the table properties.
-  
```

Comment:
Why need to append one more rows at here ? please remove all needless format changes in this PR.